util/netcdf_generator.py

```diff
@@ -59,13 +59,16 @@ def _filter_params(self, ds, params):
         default_params = ['time', 'deployment', 'id', 'lat', 'lon', 'quality_flag']
 
         for param in params:
-            if param not in ds.data_vars:
+            # look for param in both data_vars and coords (13025 AC2)
+            if param not in ds.data_vars and param not in ds.coords:
                 missing_params.append(param)
             else:
                 params_to_filter.append(param)
 
-        if missing_params:
+        # only complain about missing parameters for directly requested data stream
```

Comment:
only `warn` ...

Reply:
will update to make the comment sound less harsh.